Blockzilla/SettingsViewController.swift

```diff
@@ -593,3 +602,66 @@ extension SettingsViewController: SearchSettingsViewControllerDelegate {
         (tableView.cellForRow(at: IndexPath(row: 0, section: 0)) as? SettingsTableViewAccessoryCell)?.accessoryLabelText = engine.name
     }
 }
+
+extension SettingsViewController: INUIAddVoiceShortcutViewControllerDelegate {
+    @available(iOS 12.0, *)
+    func addVoiceShortcutViewController(_ controller: INUIAddVoiceShortcutViewController, didFinishWith voiceShortcut: INVoiceShortcut?, error: Error?) {
+        controller.dismiss(animated: true, completion: nil)
+    }
+    
+    @available(iOS 12.0, *)
+    func addVoiceShortcutViewControllerDidCancel(_ controller: INUIAddVoiceShortcutViewController) {
+        controller.dismiss(animated: true, completion: nil)
+    }
+    
+    @available(iOS 12.0, *)
+    func manageSiri(for activityType: SiriShortcuts.activityType) {
+        INVoiceShortcutCenter.shared.getAllVoiceShortcuts { (voiceShortcuts, error) in
+            DispatchQueue.main.async {
+                guard let voiceShortcuts = voiceShortcuts else { return }
+                let foundShortcut = voiceShortcuts.filter { (attempt) in
+                    attempt.__shortcut.userActivity?.activityType == activityType.rawValue
```

Comment:
I think the double-underscore is incorrect here

Reply:
Interesting, looks like it changed from Beta 5 -> Beta 6.